java/java-hello-world/.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+// launch.json contains information needed for the debugger to attach to  processes
```

Comment:
.vscode shouldn't be checked in.  .gitignore at root?

Reply:
.vscode folder is needed - there are configurations that are integral part of the templates.